lisa/schema.py

```diff
@@ -580,6 +580,51 @@ def _generate_min_capability(self, capability: Any) -> Any:
         return min_value
 
 
+@dataclass_json()
+@dataclass()
+class NvmeOptionSettings(FeatureSettings):
+    type: str = "NVME"
+    nvme_disk_count: search_space.CountSpace = field(
```

Comment:
It's in nvme already, the name can be `disk_count`.

Reply:
updated